src/app/Fake.Testing.SonarQube/SonarQube.fs

```diff
@@ -48,7 +55,7 @@ module Fake.Testing.SonarQube
       
       let args = 
         match call with
-        | Begin -> "begin /k:\"" + parameters.Key + "\" /n:\"" + parameters.Name + "\" /v:\"" + parameters.Version + "\" " + setArgs + cfgArgs
+        | Begin -> "begin /k:\"" + parameters.Key + "\" /n:\"" + parameters.Name + "\" /v:\"" + parameters.Version + "\" " + setArgs + cfgArgs + orgArgs
```

Comment:
Ideally we would use the Arguments-API instead of string concatenation. Other modules like `DotNet.Cli` already use it. It also eases writing tests without actually starting anything.
Any interest in pushing this into that direction? The docs are here:  https://fake.build/core-process.html